src/main/java/com/google/sps/placeGuide/repository/PlaceGuideRepository.java

```diff
@@ -15,7 +15,7 @@
   // Finds all of the user's created place guides.
   public List<PlaceGuide> getCreatedPlaceGuides(String creatorId);
 
-  // Finds all of the user's created place guides that are public 
+  // Finds all of the user's created place guides that are public
```

Comment:
javadoc format. Here and below

```suggestion
  /** Finds all of the user's created place guides that are public
```

Reply:
Done.